config/module.go

```diff
@@ -42,7 +41,7 @@ type Module struct {
 	Environment map[string]string `json:"env,omitempty"`
 
 	// FirstRunTimeout is the timeout duration for the first run script.
-	FirstRunTimeout time.Duration `json:"first_run_timeout,omitempty"`
+	FirstRunTimeout utils.Duration `json:"first_run_timeout,omitempty"`
```

Comment:
`time.Duration` fields cannot be JSON unmarshaled. the pattern we've used to get around this is creating a parallel struct where all `time.Duration` fields are strings. this parallel struct's custom unmarshal method then parsed these fields using `time.ParseDuration`. this works, but requires us to always maintain a parallel struct. I think a better approach is to have a custom duration field that supports (un)marshaling, which is what I implement in this PR.